src/CrystalIntrinsicAction.cpp

```diff
@@ -31,7 +31,7 @@ CrystalIntrinsicAction::~CrystalIntrinsicAction()
 // This function is called at the begining of event
 void CrystalIntrinsicAction::GeneratePrimaries( G4Event* anEvent )
 {
-  if ( m_crystalType != "LSO" && m_crystalType != "LYSO" )
+  if ( m_crystalType != "LSO" && m_crystalType != "LYSO" && m_crystalType != "NaI" && m_crystalType != "BGO" && m_crystalType != "CsF")
```

Comment:
Why do you allow generation of a Lu176 background from crystals that do not contain Lu176 (or other radioisotopes?)

Reply:
It was a mistake... Fixed now